core/field_image.js

```diff
@@ -37,8 +37,8 @@ goog.require('goog.math.Size');
 /**
  * Class for an image on a block.
  * @param {string=} src The URL of the image. Defaults to an empty string.
- * @param {!(string|number)} width Width of the image.
- * @param {!(string|number)} height Height of the image.
+ * @param {!(string|number)=} width Width of the image.
+ * @param {!(string|number)=} height Height of the image.
  * @param {string=} opt_alt Optional alt text for when block is collapsed.
  * @param {Function=} opt_onClick Optional function to be called when the image
```

Comment:
Can't have required fields after the optional field (src). 
Alternately we can make src required. 

If we decide to go with this change we should probably have defaults for height and width.

Reply:
Currently not passing a src is still /technically/ invalid (or at least bad behavior). The thing is that it just shows a missing image icon instead of throwing an error.

I think it would be better to make src required and throw errors if it is not provided correctly (null, undefined, empty string).

Reply:
That makes sense. I'll change it back to required, and add the check

Reply:
Looks perfect thank you!